docs/build/identity.rs/v0.5.0/docs/specs/didcomm/overview.mdx

```diff
@@ -3,10 +3,6 @@ title: IOTA DIDComm Specification
 sidebar_label: Overview
 ---
 
-import Announcement from '../../../src/partials/_announcement.mdx';
-
-<Announcement />
-
 :::info
 
 The IOTA DIDComm Specification is in the RFC phase and may undergo changes. Suggestions are welcome at [GitHub #464](https://github.com/iotaledger/identity.rs/discussions/464).
```

Comment:
Could we add this to a banner as well?

Reply:
That would display that banner in all docs of this Identity version. Not sure if that's intended here?

Reply:
I see, so in that case we'd like to support multiple banners right? I can do that for sure. Would you like to create an issue for that too?

Reply:
Tracked by issue #1107